### DISCUSSION TITLE
How to change the file and get file name in source ?

### DISCUSSION THREAD

**Discussion by dream811** [obs-websocket-community-projects/obs-websocket-js#254] [Q&A]
I am going to controller using obs-websocket-js
for example:import media file, and create&change media in source and start, stop, pause of the scene of streaming now.
I want to stop the video or replace it with another video  during broadcasting .
I hope to see all sample codes about this socket request and event.
thanks!
How to get  and set file name via GetSourceSettings and SetSourceSettings.
please help me!
![image](https://user-images.githubusercontent.com/45660446/62445611-3c39b080-b793-11e9-92ce-a1ea2b9c91cf.png)

**Comment by marckraw** [comment]
@dream811 maybe this is something that can help you: https://github.com/haganbmj/obs-websocket-js/issues/164#issuecomment-717838452
